papis/utils.py

```diff
@@ -71,9 +71,22 @@ def general_open(file_name: str,
         subprocess.call(cmd)
     else:
         logger.debug("Not waiting for process to finish")
-        subprocess.Popen(
-            cmd, shell=False,
-            stdin=None, stdout=None, stderr=None, close_fds=True)
+        popen_kwargs = {
+            "shell": False,
+            "stdin": None,
+            "stdout": None,
+            "stderr": None,
+            "close_fds": True
+        }
+        # Tell subprocess to detach the process.
+        if sys.platform == "win32":
+            popen_kwargs["creationflags"] = subprocess.DETACHED_PROCESS
+            popen_kwargs["creationflags"] |= subprocess.CREATE_NEW_PROCESS_GROUP
+        else:
+            cmd.insert(0, "nohup")
+            popen_kwargs["stdout"] = subprocess.DEVNULL
+            popen_kwargs["stderr"] = subprocess.DEVNULL
```

Comment:
Why set these to `DEVNULL` only here?

Reply:
I thought they are Unix only, but [the doc](https://docs.python.org/3/library/os.html#os.devnull) says, it's None on Windows. I'll put it to the default popen_kwargs.